src/neo/Cryptography/MPT/MPTNode.cs

```diff
@@ -8,8 +8,10 @@ namespace Neo.Cryptography.MPT
     public abstract class MPTNode
     {
         private UInt256 hash;
-
+        public long Reference = 1;
```

Comment:
Only LeafNode need this?

Reply:
All node need this. If two contracts have same address and balances, they share same child tree.